runtime/common/src/crowdloan/migration.rs

```diff
@@ -29,6 +29,11 @@ pub mod crowdloan_index_migration {
 	pub fn pre_migrate<T: Config>() -> Result<(), &'static str> {
 		// `NextTrieIndex` should have a value.
 		generate_storage_alias!(Crowdloan, NextTrieIndex => Value<FundIndex>);
+
+		generate_storage_alias!(
+			Slots,
+			Leases<T: Config> => Map<(Twox64Concat, ParaId), Vec<Option<(T::AccountId, BalanceOf<T>)>>>
```

Comment:
Does this work? It looks like the macro takes `(Key, Hasher)` but here you have `(Hasher, Key)`. See: https://github.com/paritytech/substrate/blob/77c15d2546276a865b6e8f1c5d4b1d0ec1961e72/frame/support/src/lib.rs#L185 (I am not competent in macros so def could be wrong)

Reply:
hmm something to investigate for sure, but i see my syntax in contracts

<img width="663" alt="image" src="https://user-images.githubusercontent.com/1860335/159777722-f062cac1-901d-4b11-afd7-f2dc9098e0fa.png">

Reply:
fixed it: https://github.com/paritytech/substrate/pull/11102